curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3477ae9675db8bb7655b30.md

```diff
@@ -18,6 +18,13 @@ const hasH1 = new __helpers.CSSHelp(document).getStyle('h1');
 assert(hasH1);
 ```
 
+You should not add a new `style` tag. Add the new CSS rules to the existing `style` tag.
```

Comment:
Agreed, the "element" is everything and a "tag" is either the opening or closing tag itself. So you would either say "You should not add new style tags" or "You should not add a new style element." Likewise, either "Add the new CSS rules between the existing style tags" or "Add the new CSS rules to the existing style element".

In the real world we often use tag and element interchangeably. But it seems the convention here is to make this distinction.

Reply:
Right, my point is that we both do not want any new style elements and we do not want any new style tags. The question specifically states not to add any new style elements, which is the common error we are trying to prevent, but we are checking for that via checking for tags (which makes sense). I'm just making sure we use the word that we most want to use here.

Reply:
From translation prespective, tags are hard to translate (to Arabic, at least) so unless tags are checked for in the test for example a regex test, I would prefer to not use them.

Reply:
```suggestion
You should not add a new `style` element. Add the new CSS rules to the existing `style` element.
```

```suggestion
You should not add new `<style>` tags. Add the new CSS rules between the existing `<style>` tags.
```

Or mix and match. These seems more correct grammatically.

Reply:
We should use the language matching the test; is the test asserting tags or elements?